Client/vite.config.js

```diff
@@ -1,22 +1,20 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 
+const proxyTarget = 'http://localhost:1113';
```

Comment:
Ah, I see that you are using a proxyTarget! Is this what you would use to update for your production env instead?